hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/SparkClientFunctionalTestHarness.java

```diff
@@ -164,7 +164,7 @@ public synchronized void runBeforeEach() {
       SparkConf sparkConf = conf();
       SparkRDDWriteClient.registerClasses(sparkConf);
       HoodieReadClient.addHoodieSupport(sparkConf);
-      spark = SparkSession.builder().config(sparkConf).getOrCreate();
+      spark = SparkSession.builder().config(sparkConf).enableHiveSupport().getOrCreate();
```

Comment:
instead of changing this for all tests, can you override `org.apache.hudi.testutils.providers.SparkProvider#conf()` in your specific test class to pass in the configs you need for your testcase?

Reply:
In enableHiveSupport function, spark also check if hiveClassesArePresent, if not then throw a exception, do you mean I should pass config directly and ignore this logic?